docs/reference/kusk-cli.md

```diff
@@ -105,39 +105,63 @@ the upstream service, namespace and port to the flag values passed in respective
 This is enough to get you started.
 
 If the `x-kusk` extension is already present, it will override the the upstream service, namespace and port to the flag values passed in respectively
-and leave the rest of the settings as they are
+and leave the rest of the settings as they are.
+
+You must specify the name of the envoyfleet you wish to use to expose your API. This is because Kusk Gateway could be managing more than one.
+In the future, we will add the notion of a default envoyfleet which kusk gateway will use when none is specified.
```

Comment:
Can we add a hint on the name of the default envoy-fleet created by the installer?